observer.go

```diff
@@ -46,21 +46,56 @@ func initTLS() *tls.Config {
 
 	if len(config.Aerospike.RootCA) > 0 {
 		// Try to load system CA certs and add them to the system cert pool
-		caCert, err := ioutil.ReadFile(config.Aerospike.RootCA)
-		if err != nil {
-			log.Fatalf("FAILED: Adding server certificate `%s` to the pool failed: %s", config.Aerospike.RootCA, err)
-		}
+		caCert := readCertFile(config.Aerospike.RootCA)
 
 		log.Infof("Adding server certificate `%s` to the pool...", config.Aerospike.RootCA)
 		serverPool.AppendCertsFromPEM(caCert)
 	}
 
 	var clientPool []tls.Certificate
 	if len(config.Aerospike.CertFile) > 0 || len(config.Aerospike.KeyFile) > 0 {
-		// client Cert
-		cert, err := tls.LoadX509KeyPair(config.Aerospike.CertFile, config.Aerospike.KeyFile)
+
+		// Read Cert and Key files
+		certFileBytes := readCertFile(config.Aerospike.CertFile)
+		keyFileBytes := readCertFile(config.Aerospike.KeyFile)
+
+		// Decode PEM data
+		keyBlock, _ := pem.Decode([]byte(keyFileBytes))
```

Comment:
Shouldn't you check errors here?

Reply:
There's a check right after this code to see if `keyBlock` is `nil`. `keyBlock` will be `nil` if it's unable to find or decode the PEM data in the input `keyFileBytes`.
https://golang.org/pkg/encoding/pem/#Decode